Stack/Opc.Ua.Core/Types/Encoders/IDecoder.cs

```diff
@@ -173,9 +173,13 @@ public interface IDecoder
         ExtensionObject ReadExtensionObject(string fieldName);
 
         /// <summary>
-        ///  Reads an encodeable object from the stream.
+        /// Reads an encodeable object from the stream.
         /// </summary>
-        IEncodeable ReadEncodeable(string fieldName, System.Type systemType);
+        /// <param name="fieldName">The encodeable object field name</param>
+        /// <param name="systemType">The system type of the encopdeable object to be read</param>
+        /// <param name="encodeableTypeId">The TypeId for the <see cref="IEncodeable"/> instance that will be read.</param>
+        /// <returns>An <see cref="IEncodeable"/> object that was read from the stream.</returns>
+        IEncodeable ReadEncodeable(string fieldName, System.Type systemType, ExpandedNodeId encodeableTypeId = null);
```

Comment:
looks like a breaking change to IDecoder interface. We need to bump up version json to 366.

Reply:
or is a caller able to consume it without code change? Then we can stick with 365

Reply:
No breaking change since since default value is set  `encodeableTypeId = null`

Reply:
@AlinMoldovean, then it looks good to me.